front/components/AppLayout.tsx

```diff
@@ -67,7 +67,7 @@ export default class AppLayout extends React.Component<AppLayoutProps, AppLayout
       return (
         <div style={{ color: 'white' }}>
           <Link href="/login">
-            <a><Button ghost>Faculty login</Button></a>
+            <a><Button ghost>Faculty Login</Button></a>
```

Comment:
Sorry, I am following the sentence case to keep the consistency among the page. So this stays as "Faculty login"
Reference: https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98